state/cluster/root_block.go

```diff
@@ -2,15 +2,29 @@ package cluster
 
 import (
 	"fmt"
+	"regexp"
 
 	"github.com/onflow/flow-go/model/cluster"
 	"github.com/onflow/flow-go/model/flow"
 )
 
+// ClusterChainPrefix is the prefix used for all Collection Cluster chains.
+// Clusters change every epoch, resulting in a new ChainID every epoch.
+// Cluster ChainIDs are generated by [CanonicalClusterID] and verified by [ChainID.IsCanonicalClusterID].
```

Comment:
function `IsCanonicalClusterID` is not attached to the struct `ChainID`

```suggestion
// Cluster ChainIDs are generated by [CanonicalClusterID] and verified by [IsCanonicalClusterID].
```